kernel/opaqueproof.ml

```diff
@@ -57,15 +53,14 @@ let not_here () =
   CErrors.user_err Pp.(str "Cannot access opaque delayed proof")
 
 let create dp cu tab =
-  let hcons (c, u) =
-    let c = Constr.hcons c in
-    let u = match u with
-    | PrivateMonomorphic u -> PrivateMonomorphic (Univ.hcons_universe_context_set u)
-    | PrivatePolymorphic (n, u) -> PrivatePolymorphic (n, Univ.hcons_universe_context_set u)
+  let map (c, u) =
```

Comment:
maybe this function should be called `to_disk` or something like that. `map` is not really clear.